pkg/asset/tls.go

```diff
@@ -31,13 +36,13 @@ func newTLSAssets(altNames tlsutil.AltNames) ([]Asset, error) {
 		return assets, err
 	}
 
-	kubeletKey, kubeletCert, err := newKubeletKeyAndCert(caCert, caKey)
+	kubeletKey, kubeletCert, err := newKubeletKeyAndCert(caCert, caPrivKey)
```

Comment:
That is correct, as per the current implementation. I think we could handle passphrase protected keys, but it's not being done currently.